src/canon/Environment.jl

```diff
@@ -66,7 +66,7 @@ end)
 end)
 
 @docapode("Tracer"
-          ,"https://google.com"
+          ,""
```

Comment:
This is from 

https://clima.github.io/OceananigansDocumentation/stable/physics/buoyancy_and_equations_of_state/#Buoyancy-tracer